mptt/static/mptt/draggable-admin.js

```diff
@@ -402,7 +402,11 @@ django.jQuery(function($){
                 expandOrCollapseNode(treeNode(storedNodes[i]));
             }
         } else {
-            collapseTree();
+            if (DraggableMPTTAdmin.expandTreeByDefault) {
+                expandTree();
```

Comment:
I don't think the expandTree call is necessary in this case since the tree should be fully expanded by default.

Maybe you could rename the feature to `collapse_tree_by_default = True` and simply do `DraggableMPTTAdmin.collapseTreeByDefault && collapseTree()` or something.

I don't care much either way though, I just think that since we have to do a thing for collapsing the tree and nothing for expanding it, using "collapse" to name the feature might be clearer.